package.json

```diff
@@ -34,12 +34,10 @@
   "lint-staged": {
     "*.{js,ts,tsx}": [
       "eslint --fix",
-      "prettier --write",
-      "git add"
```

Comment:
I pulled this down and tested locally. Everything worked great upon install. It does look like `git add` has been added automatically for lint staged so was no longer needed, ref: https://github.com/lint-staged/lint-staged/blob/main/CHANGELOG.md#breaking-changes-4